### PR TITLE
Bs 13763 missing iterator

### PR DESCRIPTION
**Pull request by azara**

### CODE REVIEW

**Comment by rbioteau** [comment]
Can you squash your commit please ?

File: bundles/plugins/org.bonitasoft.studio.engine/src-test/java/org/bonitasoft/studio/engine/export/builder/EngineFlowElementBuilderTest.java [hunk 1]
```diff
@@ -181,6 +192,26 @@ public void testAddContext(){
...
+        final Pool pool = PoolBuilder.aPool().havingElements(taskB).build();
+        final MainProcess mainProcess = MainProcessBuilder.aMainProcess().build();
+        mainProcess.getElements().add(pool);
+        mainProcess.getDatatypes().add(StringDataTypeBuilder.aStringDataType().build());
+        flowElementSwitch.addContext(taskBuilder, (Task) pool.getElements().get(0));
+
+        final ArgumentCaptor<Expression> argument = ArgumentCaptor.forClass(Expression.class);
+        verify(taskBuilder).addContextEntry(eq("pData_ref"), argument.capture());
+        assertThat(argument.getValue().getName()).isEqualTo("pData");
+    };
```
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.engine/src-test/java/org/bonitasoft/studio/engine/export/builder/EngineFlowElementBuilderTest.java]: semi column to remove

**Comment by azara** [comment]
Why ?

**Comment by rbioteau** [comment]
You have two commit with the same message.

**Pull request merged**